runtime/router.go

```diff
@@ -234,6 +241,10 @@ func (router *HTTPRouter) handleMethodNotAllowed(
 	w http.ResponseWriter,
 	r *http.Request,
 ) {
+	scopeTags := map[string]string{scopeTagEndpoint: router.notFoundEndpoint.EndpointName, scopeTagHandler: router.notFoundEndpoint.HandlerName}
```

Comment:
Wrong endpoint and handler name tag, should be methodNotAllowed

Reply:
fixed